log2ram

```diff
@@ -2,37 +2,34 @@
 
 HDD_LOG=/var/log.hdd/
 RAM_LOG=/var/log/
+LOG2RAM_LOG="${HDD_LOG}log2ram.log"
 SIZE=40M
 USE_RSYNC=false
 
+sync () {
+    if [ "$USE_RSYNC" = true ]; then
+        rsync -aXWv --delete --links $HDD_LOG $RAM_LOG 2>&1 | tee -a $LOG2RAM_LOG
```

Comment:
`tee`'s there just in case someone's testing it from the command line.

Also, I realized I combined sync-from-disk (in start) and sync-to-disk (in stop/write).  I'll push a patch in a few minutes that fixes that unfortunate mistake.